incubator/hnc/README.md

```diff
@@ -129,3 +129,25 @@ milestones are defined:
 
 Non-coding tasks are also tracked in the [HNC
 project](https://github.com/kubernetes-sigs/multi-tenancy/projects/4).
+
+## Releasing
+
+[comment]: # (TODO: pull the code directly from Github)
+1.Check out the code to the version that you want to release and run the 
+following command in `incubator/hnc`.
+```bash
+make release
+```
+It will use Cloud Build (on your GCP account)
+to build the container image remotely while building YAMLs locally.
+
+2.Make sure you have logged into your Docker ID which has the access to MTWG [Docker Hub](https://cloud.docker.com/u/multitenancy/repository/docker/multitenancy/tenant).
```

Comment:
With the access granted, we should point to gcr.io/k8s-staging-multitenancy

Reply:
As discussed in the Github [issue](https://github.com/kubernetes-sigs/multi-tenancy/issues/43), gcr.io is blocked in China, so we decided to use the Docker Hub repo owned by MTWG for better availability. This is the link to that Docker Hub.

Reply:
We were able to confirm that other asia.gcr.io were accessible and we'd prefer to have the authentication handled for us via k8s hooks. I took a look at the other sigs and almost all are using gcr.io/k8s-staging-* so it'd seem like an anti-pattern not to IMO.

@adrianludwin thoughts?

Reply:
Given that asia.gcr.io is available in China, yes, we should go with that. Thanks Ryan.